QSB/Utility/DebugLog.cs

```diff
@@ -9,7 +9,20 @@ public static class DebugLog
     {
         public static void ToConsole(string message, MessageType type = MessageType.Message)
         {
-            QSB.Helper.Console.WriteLine(message, type);
+            var console = (ModSocketOutput)QSB.Helper.Console;
+            var method = console.GetType()
+                .GetMethods(System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance)
+                .Last(x => x.Name == "WriteLine");
+            var callingType = GetCallingType(new StackTrace());
+            if (callingType == "DebugLog")
+            {
+                callingType = GetCallingType(new StackTrace(), 2);
+            }
+            if (callingType == "DebugLog")
+            {
+                callingType = GetCallingType(new StackTrace(), 3);
+            }
+            method.Invoke(console, new object[] { type, message, callingType });
```

Comment:
Sorry alek 😅

Reply:
Seems like OWML should support another overload for WriteLine?

Reply:
I suggested that before, but you didn't seem too keen on mods being able to spoof log location easily